src/ResourceManager/Profile/Commands.Profile/Context/GetAzureRMContext.cs

```diff
@@ -26,6 +27,26 @@ namespace Microsoft.Azure.Commands.Profile
     [OutputType(typeof(PSAzureContext))]
     public class GetAzureRMContextCommand : AzureRMCmdlet
     {
+        /// <summary>
+        /// Gets the current default context.
+        /// </summary>
+        protected override AzureContext DefaultContext
+        {
+            get
+            {
+                if (DefaultProfile == null || DefaultProfile.Context == null)
+                {
+                    WriteError(new ErrorRecord(
+                        new PSInvalidOperationException("Run Login-AzureRmAccount to login."),
+                        string.Empty,
+                        ErrorCategory.AuthenticationError,
+                        null));
+                }
+
+                return DefaultProfile.Context;
```

Comment:
This will still throw an exception if DefaultProfile is null.  I think the right way to handle this is do have the DefaultContext return null, and write the error inside ExecuteCmdlet